web/src/types/index.ts

```diff
@@ -0,0 +1,7 @@
+export enum Tabs {
```

Comment:
Naming `Tabs` seems a little bit too generic, what do you think?

Reply:
Thanks. I have renamed it to `TabsType` as we have a component called `Tabs`